src/utils/average.ts

```diff
@@ -0,0 +1,66 @@
+import {Auto, PreMatch, Settings, Teleop} from "../ContextProvider";
+import supabase from "../Supabase/supabaseClient";
+import findClimbPoints from "./findPoints";
+
+const average = (old: number, newVal: number, denom: number) => {
+    return (old + newVal) / denom;
```

Comment:
If `old` is the old average then I would expect this to be `(old * (denom - 1) + newVal) / denom`

If old is the old sum, then carry on

Reply:
old is the old average. I completely overlooked that, thanks for the catch.